Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -338,11 +338,11 @@ public class ClusterSystem: DistributedActorSystem, @unchecked Sendable {
         /// Starts plugins after the system is fully initialized
         self.settings.plugins.startAll(self)
 
-        self.log.info("Actor System [\(self.name)] initialized.")
+        self.log.info("ClusterSystem [\(self.name)] initialized.")
         if settings.enabled {
-            self.log.info("Actor System Settings in effect: Cluster.autoLeaderElection: \(self.settings.autoLeaderElection)")
-            self.log.info("Actor System Settings in effect: Cluster.downingStrategy: \(self.settings.downingStrategy)")
-            self.log.info("Actor System Settings in effect: Cluster.onDownAction: \(self.settings.onDownAction)")
+            self.log.info("Setting in effect: Cluster.autoLeaderElection: \(self.settings.autoLeaderElection)")
+            self.log.info("Setting in effect: Cluster.downingStrategy: \(self.settings.downingStrategy)")
+            self.log.info("Setting in effect: Cluster.onDownAction: \(self.settings.onDownAction)")
```

Comment:
A few leftovers :) no big deal

Reply:
oops 👍

Reply:
No worries, there's tons of them, I'm sure we'll bump into other similar things